BitWatch/HomeView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  HomeView.swift
+//  BitWatch
+//
+//  Created by Sean Dunford on 1/11/15.
+//  Copyright (c) 2015 Razeware LLC. All rights reserved.
+//
+
+import UIKit
+import BitWatchKit
+
+public class HomeView: UIView {
+    var height: CGFloat!
```

Comment:
I'm not clear on the ! and ? difference they both indicate optional vars but not sure of the exact difference.